read.go

```diff
@@ -298,7 +301,7 @@ func (d *indexData) readNgrams(toc *indexTOC) (map[ngram]simpleSection, error) {
 	return ngrams, nil
 }
 
-func (d *indexData) readFileNameNgrams(toc *indexTOC) (map[ngram][]uint32, error) {
+func (d *indexData) readFileNameNgrams(toc *indexTOC) (map[ngram][]byte, error) {
```

Comment:
I suspect it's because they were already decoding everything else that was in the TOC into an in-memory representation. It's not _completely_ clear that filenameNgrams actually needs to stay resident in memory regardless-- changing it to filenameNgramOffsets might make sense.